src/main/scala/edu/ie3/simona/model/participant2/StorageModel.scala

```diff
@@ -333,13 +333,11 @@ object StorageModel {
         .getValue
         .doubleValue
     )
-    def getInitialState(eStorage: Energy, config: StorageRuntimeConfig)(
-        tick: Long,
-        simulationTime: ZonedDateTime,
-    ): StorageState = {
-      val initialStorage = eStorage * config.initialSoc
-      StorageState(storedEnergy = initialStorage, tick)
-    }
+    val getInitialState: (Long, ZonedDateTime) => StorageState =
+      (tick, _) => {
+        val initialStorage = eStorage * config.initialSoc
+        StorageState(storedEnergy = initialStorage, tick)
```

Comment:
Maybe renaming this to intitialStoredEnergy would make this easier to understand directly

Reply:
Fair point